bundle/manifests/cluster-logging.v4.6.0.clusterserviceversion.yaml

```diff
@@ -249,6 +249,12 @@ spec:
           - consoleexternalloglinks
           verbs:
           - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - "*"
```

Comment:
Same

Reply:
that is something to fix in the `manifest` dir, the bundle simply pulls from there.